build.gradle

```diff
@@ -106,6 +100,6 @@ license {
 }
 
 // Gradle version used for generating the Gradle wrapper
-task wrapper(type: Wrapper) {
-  gradleVersion = '2.10'
+wrapper {
+  gradleVersion = '5.6'
```

Comment:
You bumped the wrapper to `5.6.2` but bumped it to `5.6` here, not sure why? I'd prefer keeping build script changes outside structural changes for performers. I understand there may be some IDE conflicts these days with running Java 11 environments and such an old Gradle version, but that's why it'd be good as a *separate* PR.

Reply:
OK, I'm making a separate PR with build file changes